code/game/objects/items/stacks/stack.dm

```diff
@@ -258,9 +258,7 @@
 
 //creates a new stack with the specified amount
 /obj/item/stack/proc/split(var/tamount, var/force=FALSE)
-	if (!amount)
-		return null
-	if(uses_charge && !force)
+	if (!can_split() || !amount || (uses_charge && !force))
```

Comment:
Shouldn't `can_split()` already check `amount`?

Reply:
Well, when I thought about it, it was just a capability check. Aka "can you ever split a stack of this type". But I guess with there being a split proc I could always just merge the whole conditional statement into it as well?